.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+ - "6"
+addons:
+  firefox: latest
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
Should we be adding a `sleep` statement here to avoid test flake due to xvfb not starting quickly enough? https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI